2-types-values-variables/scope.js

```diff
@@ -0,0 +1,42 @@
+console.log("### Global and Local scope")
+
+// Global scope
+
+function localscope (){
+  // Local Scope
+  var localVar = "Can't be used outside"
+  console.log(localVar);
+};
+
+localscope();
+
+//Uncomment to see how local variables can't be used outside its scope.
+//console.log(localVar); //Uncought Reference Error.
+
+// This refers to Global Object.
+console.log(this);
+if (this == window){
+  console.log("True")
+};
+
+// This Refers to Object.
+class animal{
+	constructor(especie){
```

Comment:
aquí hay que tener cuidado con la sangría, el espaciado debe estar en espacios en lugar de tabs y hay que mantener el mismo espacio que las líneas anteriores.

Entonces acá hay que usar 2 espacio para alinear lo que está abajo de `class animal`
y luego `this.specie` dos espacios adentro de constructor.